api/src/main/resources/application.properties

```diff
@@ -13,16 +13,21 @@
 # sources). Putting properties onto "spring.datasource.tomcat" (as you might
 # see in Spring documentation) will do nothing. See WorkbenchDbConfig and
 # CdrDbConfig.
-spring.datasource.test-on-borrow=true
-spring.datasource.test-while-idle=true
-spring.datasource.time-between-eviction-runs-millis=3600000
-spring.datasource.validation-query=SELECT 1
-spring.datasource.initial-size=5
-spring.datasource.max-idle=5
-spring.datasource.min-idle=1
+spring.datasource.type=org.apache.tomcat.jdbc.pool.DataSource
+spring.datasource.tomcat.test-on-borrow=true
+spring.datasource.tomcat.time-between-eviction-runs-millis=3600000
+spring.datasource.tomcat.validation-query=SELECT 1
+spring.datasource.tomcat.initial-size=5
+spring.datasource.tomcat.max-idle=5
+spring.datasource.tomcat.min-idle=1
+# Disable liquibase because DB migration handled by anothe gradle task.
 spring.jpa.hibernate.ddl-auto=none
+spring.liquibase.enabled=false
 spring.jpa.properties.hibernate.dialect=org.pmiops.workbench.cdr.MySQLDialect
-# See https://docs.spring.io/spring-boot/docs/1.5.3.RELEASE/reference/html/production-ready-endpoints.html
+# See https://docs.spring.io/spring-boot/docs/2.3.9.RELEASE/reference/html/production-ready-features.html
 # and https://precisionmedicineinitiative.atlassian.net/browse/RW-1478
 # Disable all spring-boot endpoints unless specifically necessary
-endpoints.enabled=false
+management.endpoints.enabled-by-default=false
+# Restore 1.x Actuator feature.
```

Comment:
Is there a way to just disable this?

Reply:
The Actuator? I am not sure what they are, and if we are using those endpoints actually. I was trying to keep 1 and 2 endpoints path the same:
https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html
If we don't need this, I think I can remove it.